org.gnome.Geary.json

```diff
@@ -62,8 +59,8 @@
             "sources": [
                 {
```

Comment:
Libgee is part of GNOME Runtime now. Should be removed

Reply:
Fixed, thanks! How can we find out about this kind of thing in the future?

Reply:
You can use `flatpak run -d --command=bash org.gnome.Sdk//3.34` to have the command line inside the container.
```
[📦 org.gnome.Sdk ~]$ ls -la /usr/share/vala/vapi/ | grep gee
-rw-r--r--   6 nfsnobody nfsnobody  37605  7 sep 01:40 gee-0.8.vapi
```

Reply:
Was hoping more for "release notes announcing changes" rather than "remember and check yet another thing" ;)